dev/tools/bin/engine_hash.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
```

Comment:
Why do we need a separate bash script for this, when update_dart_sdk.sh already knows how to calculate this? https://github.com/flutter/flutter/commit/4f8a6c30d20ff32d482680b81ba1cfb691bdf2d7#diff-f72665a0a0e8577dd1721ef722b06b17172319b82e209d262627d6e3fae5a608

Reply:
this is for frob. they are currently calculating the engine hash on their own, which is bad because it is outside our control.

Reply:
but they could use your dart script, though, right? update_dart_sdk.sh should handle the bootstrapping issue.

Reply:
they cannot because they can't run dart on borg.